package.json

```diff
@@ -52,11 +53,14 @@
     "cross-env": "^3.1.4",
     "eslint": "^7.28.0",
     "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
This package assists in reducing the level of conflict between eslint and prettier.